README.md

```
@@ -1 +1,73 @@
 # Wild Backend Calculator application
+
+### 과제 개요
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)